pymeasure/display/console.py

```diff
@@ -27,45 +27,61 @@
 import os
 import copy
 import argparse
+
 try:
-    import progressbar
-    # Check that progressbar is progressbar2
-    progressbar.streams
+    from tqdm import tqdm
```

Comment:
When having logging output with tqdm, output on console is messed up. With progressbar2 it looks better to me.